staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -693,6 +695,29 @@ message LinuxContainerConfig {
     LinuxContainerSecurityContext security_context = 2;
 }
 
+// WindowsSandboxSecurityContext holds platform-specific configurations that will be
+// applied to a sandbox.
+message WindowsSandboxSecurityContext {
+    // User name to run the container process as. If specified, the user MUST
+    // exist in the container image and be resolved there by the runtime;
+    // otherwise, the runtime MUST return error.
```

Comment:
For this I just copied to comments from the fields that already existed on `WindowsContainerSecurityContext` .There are some open issues in k/k (https://github.com/kubernetes/kubernetes/issues/92963) where users are noticing that it is not possible to run the sandbox containers as a different user and this is possible on linux today.

I don't think we can fall back to the `run_as_user` - the types different (`run_as_user` is an Int64 and `run_as_username` is a string) and users are just handled different in linux vs windows.
I'm open to suggestions here.

Reply:
@jsturtevant I saw your below comment which is related 
> This does seem confusing to me. If I set a pod as podHostProcess=true I would expect all the containers to be hostProcess containers since we have the constraint that it is all or nothing. Is this for the usecase at somepoint when we may allow some containers to be hostprocess and others not?

Yea future support for pods container both hostprocess and non-hostprocess containers is the main reason why I didn't want o block pod validation if the pod sets HostProcess=true but all containers HostProcess=false.
Although this may be confusing I don't necessarily think we should block it.
I see this as more of a quirk when using setting inheritance.

Setting HostProcess=false at the Pod level was done to help out policy authors ensure HostProcess containers do not get scheduled if not allowed.

Reply:
I take back everything I just said...
The KEP discussions and KEP do state `If PodSecurityContext.WindowsSecurityContextOptions.HostProcess = true is set to true then no container in the pod sets Container.SecurityContext.WindowsSecurityContextOptions.HostProcess = false` 
Let me update this and add some additional test cases.
Sorry I mis-remembered this!